diff --git a/node_modules/@zano-project/zano-utils-js/dist/mnemonic/seed-to-mnemonic.js b/node_modules/@zano-project/zano-utils-js/dist/mnemonic/seed-to-mnemonic.js
index 2c96dc4..a770245 100644
--- a/node_modules/@zano-project/zano-utils-js/dist/mnemonic/seed-to-mnemonic.js
+++ b/node_modules/@zano-project/zano-utils-js/dist/mnemonic/seed-to-mnemonic.js
@@ -54,7 +54,6 @@ function binaryToText(binary) {
 function getWordFromTimestamp(timestamp, usePassword) {
     const dateOffset = Math.max(timestamp - WALLET_BRAIN_DATE_OFFSET, 0);
     let weeksCount = Math.trunc(dateOffset / WALLET_BRAIN_DATE_QUANTUM);
-    console.log(weeksCount);
     if (weeksCount >= WALLET_BRAIN_DATE_MAX_WEEKS_COUNT) {
         throw new Error('SEED PHRASE needs to be extended or refactored');
     }
